### PR TITLE
Html Post Form

### PR DESCRIPTION
**Pull request by AshwinCarvalho999**
Added index.html
Added style.css
Added Linters
Created an HTML form to get the details of the user or to connect with the user using Formspree
Added .gitignore file to ignore node_modules file
Added Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+          placeholder="Email Address"
+          required
+        />
+        <textarea
+          aria-label="write your message and I'll get back to you."
+          id="message"
+          name="message"
+          placeholder="Enter text here"
+          maxlength="500"
+        ></textarea>
```
Comment (leonardodiasb) [inline index.html:228]: - [ ] Great implementation of the form. But it seems that you forgot to require the message field. Please validate the `<textarea>` so the user cannot submit the form if any input field is empty as the project requires.
 -
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @AshwinCarvalho999, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Good use of GitHub flow. ✔️
 - Style looks almost identical to Figma template. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @AshwinCarvalho999 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good design
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshwinCarvalho999** [comment]
Thanks for the feedback!

**Pull request merged**